cl/corpus_importer/management/commands/harvard_opinions.py

```diff
@@ -255,14 +267,30 @@ def parse_harvard_opinions(reporter, volume, make_searchable):
             continue
 
         # TODO: Generalize this to handle all court types somehow.
```

Comment:
Is this line obsolete?

Reply:
Maybe, maybe not.  Like the Washington Court of Appeals - is an ambiguous name - so its not quite there for everything.